### PR TITLE
Add state for players viewing game and room

### PR DESCRIPTION
**Pull request by izakfr**
Closes: #288

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -10,5 +10,8 @@
   },
   "devDependencies": {
     "lerna": "^6.0.1"
+  },
+  "dependencies": {
+    "react-router-dom": "^6.8.1"
```
Comment (benshumaker) [inline frontend/package.json:15]: There's 2 package.json files for some reason. One in frontend and one at a lower level. Most the dependencies are at the lower level. I feel like it doesn't make sense to add this here. I was already using react-router-dom at lower levels of the app.

File: frontend/packages/web/src/services/websocket/websocketActionsService.ts [hunk 2]
```diff
@@ -4,6 +4,7 @@ export enum Command {
   SUBSCRIBE = 'subscribe',
   UNSUBSCRIBE = 'unsubscribe',
   HEARTBEAT = 'heartbeat',
+  VIEW_CHANGE = 'view_change',
```
Comment (benshumaker) [inline frontend/packages/web/src/services/websocket/websocketActionsService.ts:7]: Will I need to add a route and httpaction to use this new command?

**Review by benshumaker — CHANGES_REQUESTED** [review]
Some small notes

File: frontend/package.json (hunk 1 above)
Comment (izakfr) [reply frontend/package.json]: my bad

File: frontend/packages/web/src/services/websocket/websocketActionsService.ts (hunk 2 above)
Comment (izakfr) [reply frontend/packages/web/src/services/websocket/websocketActionsService.ts]: I deleted this, was there from a previous implementation iteration

**Review by benshumaker — APPROVED** [review]

**Pull request merged**